asab/log.py

```diff
@@ -447,11 +447,12 @@ def _on_write(self):
 		self._loop.remove_writer(self._socket)
 
 		while not self._queue.empty():
-			# TODO: Handle eventual error in writing -> break the cycle and restart on write handler
 			msg = self._queue.get_nowait()
-			msg = msg.encode("utf-8")
-			self._socket.sendall(msg)
-
+			try:
+				self._socket.sendall(msg)
```

Comment:
the solution works well in seacat auth.

however when i test it by raising a random `NotImplementedError()` on this line, it seems to get stuck in a loop, spamming in `stderr` like crazy. is that acceptable?

Reply:
No, this is not OK.

Provide details about NotImplementedError -> use logging.exception(...).

Reply:
no, I meant if there is something broken in `self._socket.sendall(msg)` (as it was before this fix), then the app starts spamming `stderr` with `Error when writing to syslog '('localhost', 12345)' ****error details here****` and never stops.
which, on a second thought, is probably correct, since that will indicate an application design issue.

Reply:
There could be a counter to try to do it only 100-times and then quit.

Reply:
okay, i'll merge this MR and address this issue in a new one